### PR TITLE
Move and rename uniform_random API

### PR DESCRIPTION
**Pull request by pangyoki**
### PR types

Others

### PR changes

APIs

### Describe

- Implement `uniform_random` API under `paddle/tensor/random.py`.
- Rename `uniform_random` API in `paddle/tensor/random.py` as `uniform`.
- Create alias name of `paddle.tensor.random.uniform`：`paddle.tensor.uniform`, `paddle.uniform`

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_uniform_random_op.py [hunk 1]
```diff
@@ -472,5 +474,57 @@ def test_dtype():
...
+        ps_proc = subprocess.Popen(
+            ps_cmd.strip().split(" "),
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
+        stdout, stderr = ps_proc.communicate()
+
+        assert "Error" in str(stderr), "Error: paddle.tensor.random." \
+            "uniform_random should not be imported"
+
+
```
Comment (LutaoChu) [inline python/paddle/fluid/tests/unittests/test_uniform_random_op.py:489]: 建议新增测试，直接使用 paddle.uniform API，paddle.uniform确保可设置min，max参数，例如构造paddle.uniform([2,3], min=-5, max=5)进行测试

**Review by LutaoChu — COMMENTED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]

**Review by saxon-zh — APPROVED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/test_uniform_random_op.py (hunk 1 above)
Comment (pangyoki) [reply python/paddle/fluid/tests/unittests/test_uniform_random_op.py:489]: Done

**Review by lanxianghit — APPROVED** [review]

**Pull request merged**